js/window.js

```diff
@@ -206,4 +206,30 @@ form.addEventListener('submit', (event) => {
   if (!emailValid) {
     event.preventDefault();
   }
-});
\ No newline at end of file
+});
+/* save form data */
+const userName = form.elements.name;
+const userMail = form.elements.email;
+const userMessage = form.elements.message;
+function populateStorage() {
+  localStorage.setItem('name', form.elements.name.value);
+  localStorage.setItem('mail', form.elements.email.value);
+  localStorage.setItem('message', form.elements.message.value);
```

Comment:
- [ ] **REQUIRED:** Please, in order to be saved to local storage the data object must be turned into a string using `JSON.stringfy`